scripts/dedup_assets.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Usage:
+ *   node scripts/dedup_assets.js
+ */
+
+const fs = require('fs');
+const pushManifestPath = 'build/es6-unbundled/push-manifest.json';
+const pushManifest = require(`../${pushManifestPath}`);
+const newManifest = {};
+
+// Dedup assets already pushed by shell -
+// https://github.com/Polymer/polymer-build/issues/260
```

Comment:
https://github.com/Polymer/polymer-build/issues/260 was moved to https://github.com/Polymer/prpl-server-node/issues/41.

And https://github.com/Polymer/prpl-server-node/issues/41 was closed 4 days ago: https://github.com/Polymer/prpl-server-node/issues/41#issuecomment-340658603.

Reply:
Reopened https://github.com/Polymer/polymer-build/issues/260 since the issue wasn't addressed.